x/interchainstaking/keeper/deny_list.go

```diff
@@ -0,0 +1,62 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/quicksilver-zone/quicksilver/x/interchainstaking/types"
+)
+
+// SetZoneValidatorToDenyList sets the zone validator deny list
+func (k *Keeper) SetZoneValidatorToDenyList(ctx sdk.Context, chainID string, validator types.Validator) error {
```

Comment:
Storing the entire validator would be super inefficient, storage wise. We probably only want to store `[]string` (or maybe []bytes, as addressBytes has length 20, rather than 57-61 bytes for a bech32'd valoper), or perhaps a map[string]bool, which would involve storing more data, but able to look up the validator without iterator when we check whether a given val is on the list.